Configuration/ProcessModifiers/python/run3_primaries_for_geant4_2022_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier introduce the fix of generator/Geant4 interface
+
+run3_g4primaries_2022 = cms.Modifier()
```

Comment:
Why "2022" in the name?

Reply:
This modifier should be applied to 12_4, which is simulation for 2022 data. In 2023 we will use 13_0 and modifier will not be applied. It is possible to remove 2022 but will this removal be OK for operations?

Reply:
2022 is removed